Assets/Scripts/TakeDamageFromPlayerProjectile.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using System.Linq;
```

Comment:
nit: Remove unused using statements

```suggestion
using UnityEngine;
```